src/assets/stylesheets/screens/home.scss

```
@@ -0,0 +1,19 @@
+body.home {
+  main {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .home__survey-list {
+    $self: '.survey-list';
+
+    display: flex;
+    flex: 1;
+    align-items: center;
+    justify-content: center;
+
+    #{ $self }__blank-state {
+      padding-bottom: rem(160px);
+    }
+  }
```

```suggestion
  .home__survey-list {
    display: flex;
    flex: 1;
    align-items: center;
    justify-content: center;
  }
  
  .survey-list__blank-state {
    padding-bottom: rem(160px);
  }
```

Should we make it separate as we have just only one?
Or do you have any concerns with that? 😅